src/app/COMPONENTS/task/task.component.ts

```diff
@@ -20,8 +21,9 @@ import { Router } from '@angular/router'
 
 export class TaskComponent {
   availableTasks$ = this.taskService.getAllTasks()
+  currentUser$ = this.usersService.getUser()
```

Comment:
keep the current user state in user.service.ts so you dont have to call it a lot of times, it should only be called when a user signs in or when you go on a fresh page for the first time, then it gets stored in those 2 scenarios and can be used on any page without calling getUser() anymore